src/app/authorization/login/login.component.ts

```diff
@@ -46,8 +46,8 @@ export class LoginComponent implements OnInit, OnDestroy {
 
   ngOnInit(): void {
     this.createForm();
-    if (localStorage.getItem('registeruser')?.length) {
-      let data: any = localStorage.getItem('registeruser');
+    if (localStorage.getItem('registerUser')?.length) {
+      let data: any = localStorage.getItem('registerUser');
       this.userData = JSON.parse(data);
```

Comment:
could you please assign like JSON.parse(localstorage.getitem('registerUser) as string ) ?